File: backend/src/main/java/com/gs/starship_log_system/model/Log.java
package com.gs.starship_log_system.model;

import lombok.Builder;
import lombok.Data;
import org.springframework.data.annotation.Id;

import java.util.UUID;

@Builder
@Data
public class Log {

    @Id
    private UUID id;

    private UUID userId;

    private String logDate;

    private String posting;

    private String location;

    private String header;

    private String content;
}

File: frontend/src/Log.js
export class Log {
    constructor(id, userId, logDate, posting, location, header, content) {
        this.id = id;
        this.userId = userId;
        this.logDate = logDate;
        this.posting = posting;
        this.location = location;
        this.header = header;
        this.content = content;
    }
}